### ISSUE TITLE
Switch CI from Travis to GitHub actions

### ISSUE THREAD

**Issue by imagejan** [ilastik/ilastik4ij#89]
travis.org doesn't work any more.

In order to switch CI to GitHub Actions, the following steps are required:

* run [`github-actionify.sh`](https://github.com/scijava/scijava-scripts/blob/main/github-actionify.sh) on this repository
* temporarily add @hinerm or @ctrueden to the `ilastik` org, so that they can add the SciJava secrets used by the build script to deploy to maven.scijava.org

**Comment by k-dominik** [comment]
Hi @imagejan,

thank you again for making us aware of the switch! I have ran the scripts and pushed the result.
I also made you, @ctrueden an admin - could you please add the secrets for deployment?

Cheers
Dominik

**Comment by ctrueden** [comment]
@k-dominik Sorry for the delay in reply. Are you sure you added me to the _org_ as an admin? I do not see a banner to join the org like I should, nor did I receive an email inviting me.

**Comment by k-dominik** [comment]
Hi @ctrueden,

a - I think I added you to this repo only as an admin (and you were already part of it as a developer - I guess that's why there wasn't a notification). Just to confirm, you can't go to __Settings - secrets - actions__ here?
Or do these secrets have to be org wide? I'll add you to the org anyway - but it's probably better to add the secret to this repo only?!

Cheers
Dominik

**Comment by ctrueden** [comment]
@k-dominik I added the needed secrets to the ilastik org. This is more future-proof, in case you ever need them from any other repository later, rather than locking them into only the ilastik4ij repo.

I then removed myself from the ilastik organization, like I usually do when adding secrets to other people's orgs. But I forgot that I had already been an org member. Maybe best would be to add me as an external collaborator to the ilastik4ij repo for now?
